docs/advanced-features/new-ai-plugin-usage.md

```
@@ -1,41 +1,66 @@
+随着TinyEngine低代码平台的不断升级，AI插件迎来了全面升级。新版AI插件集成了现代化的聊天界面，提供Agent模式和Chat模式双重体验，支持上传图片生成页面，支持MCP（Model Context Protocol）工具调用能力，让AI辅助开发更加智能、强大。
 
 ## 一、功能概览
 
 新版AI插件具备以下核心功能：
 
-1. **全新UI界面**：采用TinyRobot组件库，提供更现代化的聊天体验，支持Markdown语法、支持全屏模式等
-2. **智能对话**：支持与AI进行自然语言交互，完成复杂任务
-3. **Next SDK 与 MCP工具集成**：可以调用平台插件提供的各种工具，如修改属性、修改样式、创建页面等
+### 1.1 全新Agent智能搭建模式，支持模式切换
+
+**Agent模式（智能搭建）**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)